### PR TITLE
feat: add additional vrs fields to vcf annotator

### LINKED ISSUE #167: Add additional fields to info field in VCF Annotator
- vrs_start 
- vrs_end
- vrs_alt

### PR DESCRIPTION
**Pull request by korikuzma**
Close #167 

Notes:
- One additional change I made was storing empty string in VCF field if an error occurred during translation.

### CODE REVIEW

File: src/ga4gh/vrs/extras/vcf_annotation.py [hunk 1]
```diff
@@ -172,72 +213,99 @@ def annotate(
...
+            if vrs_attributes:
+                start = str(vrs_obj.location.interval.start.value) if vrs_obj else ""
+                end = str(vrs_obj.location.interval.end.value) if vrs_obj else ""
+                alt = str(vrs_obj.state.sequence) if vrs_obj else ""
+
+                vrs_field_data[self.VRS_START_FIELD].append(start)
+                vrs_field_data[self.VRS_END_FIELD].append(end)
+                vrs_field_data[self.VRS_ALT_FIELD].append(alt)
+
+    def _record_digests(  # pylint: disable=too-many-arguments,too-many-locals
```
Comment (jsstevenson) [inline src/ga4gh/vrs/extras/vcf_annotation.py:275]: we should update this method name, maybe _get_vrs_data()

**Review by jsstevenson — CHANGES_REQUESTED** [review]
👍  tiny change

**Review by wesleygoar — APPROVED** [review]
Its great.

File: src/ga4gh/vrs/extras/vcf_annotation.py (hunk 1 above)
Comment (korikuzma) [reply src/ga4gh/vrs/extras/vcf_annotation.py]: Updated

**Review by jsstevenson — APPROVED** [review]

**Pull request merged**